plotSimFitComparison.cc

```diff
@@ -0,0 +1,521 @@
+// #include "interface/utils.h"
+
+string year[3] = {"2016","2017","2018"};
+
+std::map<int,std::vector<float>> fM_sigmas = {
```

Comment:
can we take the values from the utils.h macro so that we don't forget to update them here?